### ISSUE TITLE
not working on https://jx19qy.gpa.enetedu.com/jx2022 希望大神翻到，进度条拉满，但是不记录在数据中详见附件

### ISSUE THREAD

**Issue by lilychewfighting** [igrigorik/videospeed#957]
The vedio speed up successfully,but the datas cannot be recorded in  the background

![image](https://user-images.githubusercontent.com/110766926/183287058-528ff513-c072-4522-8c69-87c052181737.png)

**Comment by ChadBailey** [comment]
Obviously, I can't test this without a login... but I think I have an idea of what may be going on.

Can you confirm, are you saying that the completion of the course is not being recorded?
